src/components/Files.js

```diff
@@ -0,0 +1,51 @@
+import React, { useCallback } from 'react';
+import { useSelector, shallowEqual, useDispatch } from 'react-redux';
+import { Dropdown } from 'carbon-components-react';
+import File from './File';
+
+import { selectFile } from '../redux/ducks';
+
+import './Files.scss';
+
+export default function Files() {
```

Comment:
This is my personal opinion, but I find it confusing to have singular and plural of the same component name.

Perhaps instead of `Files`, this could be `FileList` or `FileSelector`.